README.md

```diff
@@ -2,13 +2,14 @@
 [![Build Status](https://www.travis-ci.org/LordLukin/LogicalImages.svg?branch=master)](https://www.travis-ci.org/LordLukin/LogicalImages) 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/a43fc0aad42c42c19499964f519d2e0e)](https://app.codacy.com/app/LordLukin/LogicalImages?utm_source=github.com&utm_medium=referral&utm_content=LordLukin/LogicalImages&utm_campaign=Badge_Grade_Settings)
 
-Program do rozwiązywania obrazków logicznych - LogicalImagesSolver 
+Program for Logigal Images Solving - LogicalImagesSolver
```

Comment:
Logigal - a typo. Maybe just Logical Images Solver? :-)

Reply:
jesteś zatrważająco szybki

Reply:
Akurat przeglądałem maile i GitHub mnie powiadomił o nowym PR

Reply:
https://app.codacy.com/app/LordLukin/LogicalImages/issues/index
Przy okazji jeśli masz czas to możesz Issues z Codacy poprawić w tym pliku :)